ext/droption/droption.h

```diff
@@ -665,7 +665,11 @@ inline bool
 droption_t<int>::convert_from_string(const std::string s)
 {
     errno = 0;
-    long input = strtol(s.c_str(), NULL, 10);
+    // If we set 0 as the base, strtol() will automatically identify the base of the
```

Comment:
I think it is best to separate out the droption (and droption test) changes into a separate PR.

Reply:
OK. I will do it.

Reply:
The new PR is #5552 .